.github/workflows/dynamoMSbuild.yml

```diff
@@ -18,7 +18,8 @@ jobs:
       run: |
         echo "***Continue with the build, Good luck developer!***"
         cd "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\"
-           .\MSBuild.exe $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.sln
+           .\MSBuild.exe $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.sln /t:Restore
```

Comment:
Is this different from nuget restore on line 16?

Reply:
On the paper it's the same @aparajit-pratap , but on my tests (local and VMs) using command line, both restores are required to resolve the assemblies references

Reply:
Does that mean it restores it twice?

Reply:
Yes, that's correct, since most of the documents recommends using dotnet restore, I guess msbuild restore could have some details and that's why this was needed